with-realm/app.json

```diff
@@ -0,0 +1,29 @@
+{
+	"expo": {
```

Comment:
you can delete app.json in this example, it's not needed. you can also delete the assets directory